common/config/rush/pnpm-lock.yaml

```diff
@@ -5709,7 +5708,7 @@ packages:
   /autoprefixer/9.8.6:
     dependencies:
       browserslist: 4.16.3
-      caniuse-lite: 1.0.30001192
+      caniuse-lite: 1.0.30001260
```

Comment:
Build was failing because it was outdated. I don't know how to update dependencies in rush properly so did it manually. 
![image](https://user-images.githubusercontent.com/36186912/134860071-147503de-ace4-4b34-9c43-6d2eb844502b.png)